doc/integrator/functionality.rst

```diff
@@ -95,10 +91,10 @@ The ``vars_<project>.yaml`` file includes variables for managing *functionalitie
     then the ``functionality`` template variable will be set to a dict with one
     key/value pair: ``"<functionality_name>": ["value1","value2"]``.
 
-Using Functionalities in Templates
-----------------------------------
+Usage in Templates
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-As explained in the :ref:`integrator_functionality_configuration` section above,
+As explained in the configuration section above,
```

Comment:
Why removing the link?

Reply:
Actually, purely for rst-technical reasons. Because the file functionality is now included by the file customisation, there is a otherwise a warning about duplicate labels - the html is generated fine anyway, but Travis will fail if I leave this warning.
For a better solution, we would need to restructure the files even more (but I would like to avoid that - take it one step at a time)